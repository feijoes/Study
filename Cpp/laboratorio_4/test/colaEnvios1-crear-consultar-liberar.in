crearColaEnvios 3
imprimirColaEnvios
liberarColaEnvios
Fin