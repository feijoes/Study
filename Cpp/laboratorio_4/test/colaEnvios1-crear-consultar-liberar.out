Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Cola de envios creada con exito.
2> 
3> La cola de envios se ha liberado exitosamente.
4> Fin.
